dist/.magento.env.yaml

```diff
@@ -262,6 +262,27 @@
 #                elasticsearch_index_prefix: magento                                                                  #
 #                elasticsearch_server_timeout: '15'                                                                   #
 #######################################################################################################################
+# ELASTICSUITE_CONFIGURATION - use this variable to retain customized ElasticSearch service settings                  #
+# between deployments. Is added under 'system/default/smile_elasticsuite_core_base_settings' system configuration.    #
```

Comment:
> Is added under 'system/default/smile_elasticsuite_core_base_settings' system configuration. 

> 
I guess, you wanted to add "_this config is added under_".
Or it may be rephrase to sometihng like `All configs provided in this configuration are nested in main ElasticSuite config section __smile_elasticsuite_core_base_settings__`

Reply:
fixed